api/lib/application/organizations/organization-controller.js

```diff
@@ -31,6 +31,12 @@ const certificationResultUtils = require('../../infrastructure/utils/csv/certifi
 const certificationAttestationPdf = require('../../infrastructure/utils/pdf/certification-attestation-pdf');
 const organizationForAdminSerializer = require('../../infrastructure/serializers/jsonapi/organization-for-admin-serializer');
 
+const certificabilityByLabal = {
+  'not-available': null,
+  eligible: true,
```

Comment:
**Question :** "eligible n'est pas entre guillemets car il est en un seul mot contrairement à "not-available" et "non-eligible", c'est ça ?

Reply:
Oui c'est pour ça 😄